app/build.gradle.kts

```diff
@@ -12,7 +12,7 @@ plugins {
 android {
     compileSdkVersion(30)
     defaultConfig {
-        applicationId = "org.jellyfin.mobile"
```

Comment:
For ATV we only append .debug on debug, I think we should mirror that behavior

Reply:
(so always include the .mobile part)

Reply:
I decided against that because the package name then gets quite long and gets a bit weird..

Reply:
While I get your point I still think it's a bad decision. "org.jellyfin" is basically our prefix while "mobile" is the name for this specific app and "androidtv" is the tv app.

The ".debug" part could be interpreted as a flavor (although not as "android flavor") which is added only for non-default builds.

Reply:
Hmm, that argument makes sense. I'm still not 100% happy with the alternative. Maybe `.dbg` or `.dev` as suffix to keep it shorter?

Reply:
We could do that but I'd like to keep this consistent with the ATV app so we'd have to change it there too.

Reply:
Appending `.debug` is pretty common from what I've seen, I think that would be fine.

Reply:
Fine, changed it.